packages/firebase_messaging/firebase_messaging/example/test/remote_message_test.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:firebase_messaging/firebase_messaging.dart';
```

Comment:
Can you move this file into https://github.com/FirebaseExtended/flutterfire/tree/master/packages/firebase_messaging/firebase_messaging_platform_interface/test 

Added tests should reside in the same package, you should just be able to move this into the directory above, though you'll need to adjust your import to use RemoteMessage from `firebase_messaging_platform_interface`